src/components/AnimeLocalForm.vue

```diff
@@ -141,6 +144,25 @@ const settingsModalOpen = ref(false)
         </div>
       </div>
 
+      <div class="sm:flex gap-4">
+        <div class="sm:text-right sm:w-32 shrink-0 mt-2 mb-1 sm:mb-0">
+          <label for="fansub" class="font-semibold">Fansubs</label>
+        </div>
+        <div class="flex-1">
+          <input
+            v-model.trim="entry.artist"
+            type="text"
+            id="fansub"
+            class="w-full"
+            placeholder="Ex. DameDesuYo/MTBB"
+          >
+          <p class="text-xs text-gray-600 mt-1.5 mb-1">
```

Comment:
I would like it if you remove this tip and mention fansubs in the tip of Animation studios itself like how in manga form authors tip mention both authors and artists

Reply:
like this?
![image](https://github.com/12ohit/aniyomi-local/assets/54363735/bb34556f-1ba4-427e-9364-73463d5c7199)

Reply:
Yup, thanks😊